Jenkinsfile_parameterized

```diff
@@ -24,6 +24,9 @@ List<LinkedHashMap<String, Object>> secrets = [
   secret('idam-sscs-systemupdate-password', 'IDAM_SSCS_SYSTEMUPDATE_PASSWORD'),
   secret('idam-sscs-oauth2-client-secret', 'IDAM_OAUTH2_CLIENT_SECRET'),
 
+  secret('robotics-email-from', 'ROBOTICS_EMAIL_FROM'),
+  secret('robotics-email-to', 'ROBOTICS_EMAIL_TO'),
```

Comment:
only needed if Jenkins needs the values for running tests. Otherwise it should be in the `bootstrap.yaml`.